src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetup.java

```diff
@@ -149,7 +148,7 @@ public static void setupCodeSyncRepo(Project project, String repoPath, String re
             ConfigFile configFile = new ConfigFile(CONFIG_PATH);
 
             if (!configFile.isRepoActive(repoPath) || isSyncingBranch) {
-                String branchName = Utils.GetGitBranch(repoPath);
+                String branchName = Utils.GetGitBranch(repoPath); //TODO handle gracefully if only file open.
```

Comment:
Probably no need to check branch if single file is opened.
```
...
if (!Utils.isIndividualFileOpen(project)) return;
String branchName = Utils.GetGitBranch(repoPath);
...
```

Reply:
Updating it.